tests/rustdoc.rs

```diff
@@ -41,14 +40,12 @@ fn rustdoc_args() {
     assert_that(p.cargo_process("rustdoc").arg("-v").arg("--").arg("--no-defaults"),
                 execs()
                 .with_status(0)
-                .with_stderr(format!("\
-[DOCUMENTING] foo v0.0.1 ({url})
-[RUNNING] `rustdoc --crate-name foo src[/]lib.rs \
-        -o {dir}[/]target[/]doc \
-        --no-defaults \
-        -L dependency={dir}[/]target[/]debug[/]deps`
-[FINISHED] dev [unoptimized + debuginfo] target(s) in [..]
-", dir = p.root().display(), url = p.url())));
+                .with_stderr_contains(&format!("[DOCUMENTING] foo v0.0.1 ({})", p.url()))
+                .with_stderr_contains("[RUNNING] `rustdoc --crate-name foo src[/]lib.rs")
+                .with_stderr_contains(&format!("-o {}[/]target[/]doc", p.root().display()))
+                .with_stderr_contains("--no-defaults")
+                .with_stderr_contains(&format!("-L dependency={}[/]target[/]debug[/]deps`", p.root().display()))
+                .with_stderr_contains("[FINISHED] dev [unoptimized + debuginfo] target(s) in [..]"));
```

Comment:
Could this instead just change to testing something like passing the `--cfg` flag? (along with tests below)

Reply:
sure